test/e2e_node/e2e_node_suite_test.go

```diff
@@ -116,9 +114,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
 	if *startServices {
 		e2es = NewE2EServices()
-		if err := e2es.Start(); err != nil {
-			glog.Fatalf("Unable to start node services: %v", err)
-		}
+		Expect(e2es.Start()).To(Succeed(), "should be able to start node services.")
```

Comment:
For my education, why is this one treated differently? Why not consistently do:

err := e2es.Start()
Expect(err).NotTo(HaveOccurred(), "should be able to start node services.")